guides/advanced_guides/asynchronous_updates.md

```diff
@@ -77,3 +77,23 @@ Failing to upload document:
   "processedAt": "2019-12-07T20:23:50.157436246Z"
 }
 ```
+
+### Killing MeiliSearch When a Task is Processing
+
+Since in MeiliSearch asynchronous task are [atomic](https://en.wikipedia.org/wiki/Atomicity_(database_systems)), killing MeiliSearch in the middle of the process does not corrupt or alter the database.
```

Comment:
```suggestion
Since in MeiliSearch asynchronous tasks are [atomic](https://en.wikipedia.org/wiki/Atomicity_(database_systems)), killing MeiliSearch in the middle of a process does not corrupt or alter the database.
```